t/socket-tcp-setoption.t

```diff
@@ -0,0 +1,725 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use lib '.';
+use t::TestCore;
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 3) - 2;
```

Comment:
We should make sure the test plan is irrelevant with the `repeat_each()` value otherwise test modes like the "check leak" mode. You can try it out yourself. cc @xiaocang

Reply:
accept